sql/2020/05_Media/big_non_custom_metrics.sql

```diff
@@ -0,0 +1,38 @@
+#standardSQL
+# non custom metrics sql that uses regexp on response bodies
+# img src vs data-uri
+# count rel=preconnect
+# video with src
+# video with source
+# figure
+# figure with figcaption
+
+SELECT client,
+  COUNTIF(has_img_data_uri) * 100 / COUNTIF(has_img_src) AS pages_with_img_data_uri_pct,
+  COUNTIF(rel_preconnect) * 100 / COUNT(0) AS rel_preconnect,
+  COUNTIF(has_video_src) * 100 / COUNT(0) AS pages_with_video_src_pct,
+  COUNTIF(has_video_source) * 100 / COUNT(0) AS pages_with_video_source_pct,
+  COUNTIF(has_figure) * 100 / COUNT(0) AS pages_with_figure_pct,
+  COUNTIF(has_figcaption) * 100 / COUNT(0) AS pages_with_figcaption_pct
```

Comment:
We should leave them as between 0 and 1, rather than 0 and 100 as Google Sheets will handle the percentage. Note [this is different to have we used to do it in 2019](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide#formatting-results)

```suggestion
  COUNTIF(has_img_data_uri) / COUNTIF(has_img_src) AS pages_with_img_data_uri_pct,
  COUNTIF(rel_preconnect) / COUNT(0) AS rel_preconnect,
  COUNTIF(has_video_src) / COUNT(0) AS pages_with_video_src_pct,
  COUNTIF(has_video_source) / COUNT(0) AS pages_with_video_source_pct,
  COUNTIF(has_figure) / COUNT(0) AS pages_with_figure_pct,
  COUNTIF(has_figcaption) / COUNT(0) AS pages_with_figcaption_pct
```

Can you do that for all the queries?